src/zenml/cli/stack_components.py

```diff
@@ -139,7 +139,7 @@ def describe_stack_component_command(name_id_or_prefix: str) -> None:
             if component_.connector:
                 # We also need the flavor to get the connector requirements
                 connector_requirements = client.get_flavor_by_name_and_type(
-                    name=component_.flavor, component_type=component_type
```

Comment:
At this point in the code, we have a `component` already, and it has the `flavor` in its `resources`. You can use it to fetch the connector requirements without making an extra hydrated `list_flavors` call.

Reply:
Taking it even one step further if you move `connector_type`, `connector_resource_type`, `connector_resource_id_attr` from `FlavorResponseMetadata` to `FlavorResponseBody`, you can do this without making any extra calls.

Reply:
I used `component_.flavor` to avoid the flavor call 👍

For the other change, I'd say this is such a niche command/use case that we don't need to optimize for, and these service connector attributes are actually quite huge in some cases, so having them in every component response does not make much sense IMO. WDYT?

Reply:
Yeah, maybe you are right. I remember that I was getting way too hyped about reducing the number of calls, got caught up a bit 😄 I think as long as we use the flavor properly, it's all good. 👍